### ISSUE TITLE
New TeSS materials

### ISSUE THREAD

**Issue by bianchini88** [korbinib/DMP-writing-workshop#16]
The new materials page, to be created after the WS at NMBU on the 20th of November, needs to be linked in the README file.

**Comment by Nazeeefa** [comment]
@bianchini88 Has this been done? :)

**Comment by bianchini88** [comment]
@Nazeeefa No, the repo needs to be in place before having a release, and we still have broken links in the README (https://github.com/ELIXIR-Norway-Training/DMP-writing-workshop/issues/18)
